@xen-orchestra/backups/writers/DeltaBackupWriter.js

```diff
@@ -52,13 +54,27 @@ exports.DeltaBackupWriter = class DeltaBackupWriter extends MixinBackupWriter(Ab
             found = found || isMergeable
           } catch (error) {
             warn('checkBaseVdis', { error })
+            Task.warning(
+              `checkBaseVdis : Error while checking existing VHD ${vdisDir}/${srcVdi.uuid} : ${error.toString()}`
+            )
```

Comment:
Since we have IDs for VM and VDI here, I think you could pass them to `Task.warning` second parameter.

However I can't remember exactly what argument name are possible. Maybe @julien-f could help here.

Reply:
The second parameter is a `data` object which can contain anything that's relevant to understand the warning.

It's usually a good idea to put data there and to keep the warning message simple and static.